hokusai/lib/docker_compose_helpers.py

```diff
@@ -22,3 +22,42 @@ def follow_extends(docker_compose_yml):
       extended_template = TemplateSelector().get(extended_template_path)
       rendered_templates.append(YamlSpec(extended_template).to_file())
     return rendered_templates
+
+def generate_compose_command(filename, default_yaml_file):
+  ''' return Docker Compose command '''
+  # this import when done globally causes circular import error
+  from hokusai.services.docker import Docker
```

Comment:
This seems like a bad pattern and is a bit confusing to follow, I think we should avoid doing this. I feel like there's two best options to change it to:

1. keep this function here, but alter it to take an extra arg, something like `compose_command`, and use that in place of `Docker.compose_command()`.
2. move `compose_command` from docker.py into this helpers file. I personally feel like it would fit well in this file but you know this code better than me!

Reply:
agreed. opted to move function to helpers.